packages/core/src/router/AbstractRouter.js

```diff
@@ -361,24 +366,29 @@ export default class AbstractRouter extends Router {
       return Promise.reject(error);
     }
 
+    const originalPath = this._getCurrentlyRoutedPath();
+    const { route } = this._getRouteHandlersByPath(originalPath);
```

Comment:
Method `_getRouteHandlersByPath` can return route property as undefined. It will crash for `route.extractParameters`.